tests/test_api.py

```diff
@@ -1908,3 +1909,19 @@ def test_do_create_prod_build_branch_required(self, osbs, branch_name):
                                            inner_template=inner_template,
                                            outer_template=outer_template,
                                            customize_conf=customize_conf)
+
+    # osbs is a fixture here
+    @pytest.mark.parametrize(('name', 'data'), [  # noqa
+        ('special-config',
+         {
+             "special.how": "very",
+             "special.type": "charm"
+         }),
+    ])
+    def test_config_map(self, osbs, name, data):
+        assert osbs.create_config_map(name, data)
```

Comment:
I put the assert in for context, to show that the `obj = {...}` and `flexmock` parts should go above it. It's saying:

- when osbs.os.create_config_map() is called, check that its parameter compares as equal to `obj` before letting execution continue ('should_call()' is a flexmock 'spy', to verify parameters as well as actually calling the method)
- call osbs.create_config_map(name, data)
- if flexmock allowed the call through we know the parameter was correct
- finally, assert that the return value from osbs.create_config_map(name,data) is truthy (as before)